integration/spark/src/main/scala/org/apache/spark/sql/secondaryindex/events/DeleteFromTableEventListener.scala

```diff
@@ -53,6 +53,12 @@ class DeleteFromTableEventListener extends OperationEventListener with Logging {
               carbonTable
                 .getDatabaseName
             }.${ carbonTable.getTableName }]")
+        } else if (!carbonTable.getIndexesMap.isEmpty) {
```

Comment:
change to 
"if (carbonTable.isIndexTable || !carbonTable.getIndexesMap.isEmpty)"

Reply:
I think that the original code has clearer error information

Reply:
ok